doc/antlr-project-testing.md

```diff
@@ -87,6 +87,8 @@ And the result of testing the entire subdirectory:
 
```

Comment:
Somethings not right in the version of the file you updated I think. The master branch has a bunch of extra information... Maybe I screwed up and did not pull this into the dev branch previously?  Here is master: https://github.com/antlr/antlr4/blob/master/doc/antlr-project-testing.md

I found it because I was looking for the part where maven still contest individual targets like:

```
$ mvn -Dtest=cpp.** test
```

Reply:
It was replaced by `mvn -Dtest=JavaRuntimeTests test` in https://github.com/antlr/antlr4/commit/f5843abaf5e7e2d763941417b04698e4b5d86b84#diff-8a1a1aeb8d87968c5aaee5b9a2c38e954b31e729767509607d65b6c87be49dcbL119 But `mvn -Dtest=java.** test` also works. I can restore the command with two asterisks if you want.